pkg/api/controllers/ledger_controller.go

```diff
@@ -12,13 +12,51 @@ import (
 	"github.com/numary/ledger/pkg/api/apierrors"
 	"github.com/numary/ledger/pkg/core"
 	"github.com/numary/ledger/pkg/ledger"
+	"github.com/numary/ledger/pkg/storage"
 	"github.com/numary/ledger/pkg/storage/sqlstorage"
 )
 
-type LedgerController struct{}
+type LedgerController struct {
+	Version       string
+	StorageDriver storage.Driver[ledger.Store]
+}
+
+func NewLedgerController(version string, storageDriver storage.Driver[ledger.Store]) LedgerController {
+	return LedgerController{
+		Version:       version,
+		StorageDriver: storageDriver,
+	}
+}
+
+type Info struct {
+	Name    string      `json:"name"`
+	Version string      `json:"version"`
+	Storage storageInfo `json:"storage"`
+}
+
+type storageInfo struct {
+	Driver     string               `json:"driver"`
+	Migrations []core.MigrationInfo `json:"migration"`
+}
+
+func (ctl *LedgerController) GetInfo(c *gin.Context) {
+	l, _ := c.Get("ledger")
 
-func NewLedgerController() LedgerController {
-	return LedgerController{}
+	var err error
+	res := Info{
+		Name:    c.Param("ledger"),
+		Version: ctl.Version,
+		Storage: storageInfo{
+			Driver: ctl.StorageDriver.Name(),
```

Comment:
Maybe not useful to set the storage driver on this endpoint, as it is available on /_info.
Not a big deal.